<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_CtoF">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>9</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius to Fahrenheit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFahr">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_FtoC">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>126</y>
      <width>140</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit to Celsius</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCel">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>85</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="C_label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </widget>
   <widget class="QLabel" name="F_label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>85</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
